src/main/java/frc/robot/Commands/AimToHub.java

```diff
@@ -0,0 +1,40 @@
+package frc.robot.Commands;
+
+import com.spikes2212.command.drivetrains.commands.DriveArcadeWithPID;
+import frc.robot.Subsystems.Drivetrain;
+import frc.robot.Subsystems.Transfer;
+
+/**
+ * <p>
+ * A command which centers the robot on one of the reflective sprites that are on the hub.
+ * </p>
+ *
+ * @author Yoel Perman Brilliant
+ */
+public class AimToHub extends DriveArcadeWithPID {
+    private static final int LIMELIGHT_PIPELINE = 1;
+
+    private final Transfer transfer;
+    private int initialPipeline;
+
+
+    public AimToHub(Drivetrain drivetrain, Transfer transfer) {
+       super(drivetrain, transfer.getLimelight()::getHorizontalOffsetFromTarget, 0, 0,
```

Comment:
you are using the getLimelight method a lot of times. Save it in a variable and use it instead of calling the getter (fix it on all of the getter calls in this class)

Reply:
Not possible at the getter in the super because you can't initialize a field before the super call, but did it for the other getters